### PR TITLE
Turn on macOS ARM64

### PR DESCRIPTION
**Pull request by BillyONeal**
Depends on (and includes) https://github.com/microsoft/vcpkg/pull/35617

### CODE REVIEW

File: scripts/azure-pipelines/azure-pipelines.yml [hunk 1]
```diff
@@ -59,6 +60,14 @@ jobs:
     vcpkgToolSha: ${{ parameters.vcpkgToolSha }}
     tripletPattern: ${{ parameters.tripletPattern }}
 
+#- template: windows/azure-pipelines.yml
```
Comment (autoantwort) [inline scripts/azure-pipelines/azure-pipelines.yml:63]:
```suggestion
- template: windows/azure-pipelines.yml
```
I think here should be no `#`

**Comment by autoantwort** [comment]
The `qtquick3d` baseline failure could be fixed by a xcode update

**Comment by autoantwort** [comment]
And you have to install `bison` via brew (the version from macOS is too old)

**Comment by BillyONeal** [comment]
> And you have to install `bison` via brew (the version from macOS is too old)

If we are expecting to get it through homebrew then vcpkg itself should be searching `/opt/homebrew/opt/bison/bin` .

**Comment by BillyONeal** [comment]
> Does vcpkg do not search in the `PATH`?

We do search that but I don't think macOS customers should need to remember to do that

**Comment by BillyONeal** [comment]
We already fix it for x64 customers here:

https://github.com/microsoft/vcpkg/blob/e3b4728c3e1c390194bcff1b1def88685af6ea2e/scripts/cmake/vcpkg_find_acquire_program(BISON).cmake#L17-L19

**Comment by BillyONeal** [comment]
Putting this part back into #35617 since that one is toast too :(

**Pull request closed without merging**